### PR TITLE
Update Localizable.strings

### PR DESCRIPTION
**Pull request by gujiaming2022**
zh-Hans.lproj Update Localizable.strings

### CODE REVIEW

File: zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -15,21 +15,21 @@
 "Latest" = "最新版";
 "Specific" = "特定版本";
 
-"Please confirm that you have adequate free storage before proceeding:" = "请确认您有足够的存储空间再进行操作:";
-"Choose the backup you'd like to restore from:" = "请选择您要恢复的备份:";
-"Are you sure that you want to restore from %@?" = "您确定要从以下备份进行恢复吗 %@?";
+"Please confirm that you have adequate free storage before proceeding:" = "请确认您有足够的存储空间再进行操作：";
```
Comment (L1ghtmann) [inline zh-Hans.lproj/Localizable.strings:18]: Is there a reason for the extra spaces added after various punctuation?

File: zh-Hans.lproj/Localizable.strings [hunk 2]
```diff
@@ -15,21 +15,21 @@
...
-"Please note:" = "请注意:";
+"Please note:" = "请注意：";
 "You have chosen to restore from a developer backup. This backup includes bootstrap packages." = "您选择了从开发者备份进行恢复。该备份包括引导包。";
 "Please confirm that you understand this and still wish to proceed with the restore" = "请确认您已理解并仍然希望继续恢复";
 
 "Confirm" = "确认";
 "Cancel" = "取消";
 
-"Tweak backup completed successfully in %@ seconds!" = "插件备份已完成，用时 %@ 秒!";
+"Tweak backup completed successfully in %@ seconds!" = "插件备份已完成，用时%@秒!";
```
Comment (L1ghtmann) [inline zh-Hans.lproj/Localizable.strings:30]: The "%@" will be replaced by a number. Is this formatting still correct?

**Review by L1ghtmann — CHANGES_REQUESTED** [review]
Hey! Thanks for the contribution. Just had a couple of questions.

File: zh-Hans.lproj/Localizable.strings (hunk 1 above)
Comment (gujiaming2022) [reply zh-Hans.lproj/Localizable.strings:18]: No, that's not a space. I will modify it to Chinese to use symbols“：”

File: zh-Hans.lproj/Localizable.strings (hunk 2 above)
Comment (gujiaming2022) [reply zh-Hans.lproj/Localizable.strings:30]: Under the habit of using Chinese, I didn't add a space between Chinese and numbers, so I deleted the space.

**Review by L1ghtmann — APPROVED** [review]

**Comment by L1ghtmann** [comment]
Thanks again!

**Comment by olivertzeng** [comment]
@gujiaming2022 哈瞜!謝謝你的翻譯!但是 zh_Hant 已經以 zh_TW 命名!應該要合併這兩個lproj檔案!
對了我看了你的翻譯後發現看來我忘記要把標點符號改為全型謝啦!

**Comment by olivertzeng** [comment]
@gujiaming2022 https://github.com/L1ghtmann/IAmLazy-Localization/pull/8

**Pull request merged**